src/main/java/com/ashkan/moviesapi/entities/Actor.java

```diff
@@ -11,7 +11,7 @@
     private Collection<ActorsInMovies> actorsInMoviesById;
 
     @Id
-    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "id")
```

Comment:
I'm not sure if you are using the jpa annotations to generate your database, but in that case it should be good to have **_nullable = false_** option configured here, in that way you are specifying this field won't accept nullable fields when trying to persist data.
The same suggestion/observation for the remaining fileds in the remaining Entity classes.

Reply:
Im working on a existing database where all ids tables are Not Null